src/recovery/mod.rs

```diff
@@ -233,13 +254,73 @@ impl Recovery {
             self.hystart.start_round(pkt_num);
         }
 
+        // Pacing: Set the pacing rate if BBR is not used
+        if self.pacing_enabled &&
+            (self.cc_algo == CongestionControlAlgorithm::CUBIC ||
+                self.cc_algo == CongestionControlAlgorithm::Reno)
```

Comment:
Sorry I need to clarify a little more. I think we simply remove checking cc algorithm names here (and then you can remove cc_algo too). Is there any issue with it? Currently we only have Reno and Cubic, but considering in the future we may add more, I think it's good to make pacing simply an option.

Reply:
AFAICT `set_pacing_rate()` is only called in this module, so I think it would be simpler to put all this code inside that method and drop the parameter.

Reply:
set_pacing_rate() will be called by congestion control algos like BBR to set the pacing rate. Since BBR is not yet implemented, and current implemented congestion control algos don't set it, its used only here.

I am adding Cubic and Reno check here because, BBR is not there yet.
Logic is, if cc_algo is BBR then it will set the pacing rate we dont need to set it here. If other algos are used we need to set the pacing rate here.

When BBR is implemented we need to add !BBR check here.

Reply:
I understand your concern but I still don't like the idea of hard coding CC names. Can we move this to each CC module (e.g. Cubic module calls set_pacing_rate() when pacing_enabled)?

Reply:
I agree that we should avoid hardcoding names here. An alternative would be to add a new function to `CongestionControlOps` like `use_generic_pacing()`, which will return `true` in the case of CUBIC and Reno, and `false` for BBR.

But generally, I also don't think this is needed at all until we actually implement BBR, so we can just remove the whole thing IMO. The comment is also confusing, as we don't even implement BBR right now.

Reply:
Thought about each CC algo sending flag to set pacing_rate external or not, but each new algo has to add code to send the flag even though it does not anything about pacing.

But anyways above idea of each CC algo sending flag is to set pacing rate or not is cleaner and comes handy for BBR too.